images/openwisp_base/Dockerfile

```diff
@@ -7,67 +7,78 @@ FROM python:3.10-slim-bullseye AS SYSTEM
 # 4. sudo
 RUN apt update && \
     apt install --yes --no-install-recommends \
-                libcairo2 apt-utils libpangocairo-1.0-0 \
-                gdal-bin gettext fping && \
+    libcairo2 apt-utils libpangocairo-1.0-0 \
+    gdal-bin gettext fping && \
     rm -rf /var/lib/apt/lists/* /root/.cache/pip/* /tmp/*
 
 RUN apt update && \
     apt install --yes --no-install-recommends \
-                gcc libpq-dev libjpeg-dev libffi-dev python3-dev \
-                python3-pip libxml2-dev libxslt1-dev zlib1g-dev g++
+    gcc libpq-dev libjpeg-dev libffi-dev python3-dev \
+    python3-pip libxml2-dev libxslt1-dev zlib1g-dev g++
 
 RUN useradd --system --password '' --create-home --shell /bin/bash \
-   --gid root --uid 1001 openwisp
+    --gid root --uid 1001 openwisp
 USER openwisp:root
 
 FROM SYSTEM AS PYTHON
 
 ENV PATH="${PATH}:/home/openwisp/.local/bin"
 ENV PYTHONPATH=/home/openwisp/.local/lib/python3.10/site-packages
+ENV EMAIL_BACKEND=djcelery_email.backends.CeleryEmailBackend
```

Comment:
Wait, why is this line here and repeated again with other env?
Does it belong here?

Wouldn't it make :
```

RUN if [ "$EMAIL_BACKEND" = "djcelery_email.backends.CeleryEmailBackend" ]; then \
    pip install --user django-celery-email~=3.0.0; \
    fi
```

always true? 

Even if we can pass the value ad-hoc to change `EMAIL_BACKEND`, I would say this should be by default not defined and we can check below:

```
if (EMAIL_BACKEND is undefined or EMAIL_BACKEND value != XYZ) 

THEN 
  do something
```

Reply:
@atb00ker what dop you mean with "repeated again with other env"? Do you mean that it's already defined on line 103?

Looks weird. As wrote in a previous comment, we could simply consider django-celery-email a base requirement and install it anyway without doing all these checks.

Reply:
> we could simply consider django-celery-email a base requirement and install it anyway without doing all these checks.

Moving forward with this because it will help in reducing complexity.